spec.txt

```diff
@@ -2626,6 +2626,15 @@ int x = 33;
 ````````````````````````````````
 
 
+```````````````````````````````` example
+<div
+>
+.
+<div
+>
+````````````````````````````````
```

Comment:
Maybe fill this “block quote” with something? I have the same comment on the 3rd new example too